packages/cli/commands/sandbox/create.js

```diff
@@ -37,18 +120,43 @@ exports.handler = async options => {
       name: sandboxName,
     })
   );
-
-  return createSandbox(accountId, sandboxName).then(
-    ({ name, sandboxHubId }) => {
-      logger.success(
-        i18n(`${i18nKey}.success.create`, {
-          name,
-          sandboxHubId,
-        })
+  let result;
+  try {
+    result = await createSandbox(accountId, sandboxName).then(
+      ({ name, sandboxHubId }) => {
+        logger.success(
+          i18n(`${i18nKey}.success.create`, {
+            name,
+            sandboxHubId,
+          })
+        );
+        return { name, sandboxHubId };
+      }
+    );
+  } catch (err) {
+    if (
+      err.error &&
+      err.error.category &&
+      err.error.category === 'MISSING_SCOPES'
```

Comment:
Added a scope specific error here:

<img width="1068" alt="Screen Shot 2022-05-06 at 2 55 46 PM" src="https://user-images.githubusercontent.com/16788677/167201636-27db0732-d013-4896-a860-834b11c015d5.png">

Reply:
Looks like we have a util for this [here](https://github.com/HubSpot/hubspot-cli/blob/master/packages/cli-lib/errorHandlers/apiErrors.js#L24). You may be able to export that and use it here